Dockerfile

```diff
@@ -1,52 +1,63 @@
-FROM elixir:1.13
+FROM almalinux:8.6
 
 ENV USER_HOME /home/docker
+ENV LANG en_US.UTF-8
 
-RUN addgroup --gid 1000 docker \
- && adduser --uid 1000 --gid 1000 --disabled-password --gecos "Docker User" --home ${USER_HOME} docker \
+# Ensure locale is UTF-8
+RUN dnf install --assumeyes \
+    glibc-langpack-en \
+    glibc-locale-source \ 
+ && localedef --force --inputfile=en_US --charmap=UTF-8 en_US.UTF-8 \
+ && echo "LANG=en_US.UTF-8" > /etc/locale.conf \
+ && dnf clean all \
+ && rm -rf /var/cache/dnf /var/cache/yum
+
+# Create non-root user
+RUN groupadd --gid 1000 docker \
+ && adduser --uid 1000 --gid 1000 --home ${USER_HOME} docker \
  && usermod -L docker
 
-# Configure apt, install updates and common packages, and clean up apt's cache
-ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get update \
- && apt-get upgrade --assume-yes \
- && apt-get autoremove --assume-yes \
- && apt-get install --assume-yes --no-install-recommends \
-    apt-utils \
-    apt-transport-https \
+# Install EPEL and base packages
+RUN dnf install --assumeyes glibc-langpack-en \
+ && dnf install --assumeyes https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm \
+ && dnf install --assumeyes dnf-plugins-core \
+ && dnf config-manager --set-enabled powertools \
+ && dnf update --assumeyes \
+ && dnf install --assumeyes \
     ca-certificates \
-    software-properties-common \
-    locales \
- && apt-get install --assume-yes --no-install-recommends \
     curl \
-    psmisc \
     git \
-    build-essential \
-    python \
     jq \
-    nodejs \
+    python36 \
```

Comment:
do we need python for something?  it does seem familiar but I don't remember at this point

Reply:
I didn't see any scripts or other parts out there with any python in them, but since python was being installed in the original dockerfile, I left it there as someone probably put it there for a reason. I can remove if that is not the case.

Reply:
oh no worries, if it was there before it's probably a requirement